### PR TITLE
cpuload add load average (using new MovingAverage filter)

### PR DESCRIPTION
**Pull request by dagar**
- add new 5s load average using MovingAverage filter
 - initial choice is a bit arbitrary, something even longer might ultimately make sense

### CODE REVIEW

**Comment by MaEtUgR** [comment]
Like this? (https://github.com/PX4/PX4-Autopilot/pull/18555) 😇

File: msg/cpuload.msg [hunk 1]
```diff
@@ -1,3 +1,6 @@
 uint64 timestamp		# time since system start (microseconds)
+
```
Comment (MaEtUgR) [inline msg/cpuload.msg:2]:
```suggestion
```

File: msg/cpuload.msg [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
 float32 load                    # processor load from 0 to 1
+float32 load_average            # processor load from 0 to 1
+
```
Comment (MaEtUgR) [inline msg/cpuload.msg:5]:
```suggestion
```

File: src/modules/load_mon/LoadMon.cpp [hunk 3]
```diff
@@ -223,7 +223,10 @@ void LoadMon::cpuload()
 	struct mallinfo mem = mallinfo();
 	cpuload.ram_usage = (float)mem.uordblks / mem.arena;
 	cpuload.load = 1.f - interval_idletime / interval;
+
```
Comment (MaEtUgR) [inline src/modules/load_mon/LoadMon.cpp:226]:
```suggestion
```

File: src/modules/load_mon/LoadMon.cpp [hunk 4]
```diff
@@ -223,7 +223,10 @@ void LoadMon::cpuload()
...
 #endif
+
+	cpuload.load_average = _cpuload_average.apply(cpuload.load);
 	cpuload.timestamp = hrt_absolute_time();
```
Comment (MaEtUgR) [inline src/modules/load_mon/LoadMon.cpp:231]:
```suggestion
```

**Review by MaEtUgR — APPROVED** [review]

**Comment by dagar** [comment]
Skip this for now (maybe we'll want it again later?).

**Pull request closed without merging**